Cabal-syntax/src/Distribution/Compat/Prelude.hs

```diff
@@ -144,6 +143,7 @@ import Prelude                       as BasePrelude hiding
     , Traversable, traverse, sequenceA
     , Foldable, foldMap
```

Comment:
Is `Foldable` here necessary since we are hiding `Foldable(..)` above?

Reply:
Fixed, thanks.